crates/ruff_python_formatter/src/expression/string/docstring.rs

```diff
@@ -572,31 +611,44 @@ impl<'src> CodeExample<'src> {
     /// the caller to perform. The typical case is a "print" action, which
     /// instructs the caller to just print the line as though it were not part
     /// of a code snippet.
-    fn add(&mut self, original: InputDocstringLine<'src>) -> CodeExampleAddAction<'src> {
+    fn add(
```

Comment:
The comment here now seems outdated considering that the function no longer returns an action.